features/search.feature

```diff
@@ -1,13 +1,36 @@
-Feature: Searching
+Feature: Searching on MAS site
   As a user visiting the website
   I want to search by keyword
   So that I can find the content I am looking for
 
-  Scenario: The search box is visible in the header
-    When I visit the home page
-    Then I should see the search box
-
-  Scenario: Search can be performed
+  Background:
     Given I am on the home page
-    When I search for something relevant
-    Then I should see search results
+
+  Scenario: Search on first result page
+    When I search the query "money"
```

Comment:
There needs to be a data setup process here. 

```gherkin
Given I am on the home page
And I have ten documents related to money
When I search for the term "money"
```

Running the tests without the data setup phase is jump which you may want to rethink?